player/lua/commands.lua

```diff
@@ -505,7 +505,9 @@ local function complete(before_cur)
             elseif tokens[first_useful_token_index + 2].text == "remove" then
                 completions = list_option_value_list(tokens[first_useful_token_index + 1].text)
             end
-        elseif first_useful_token.text == "dump-cache" then
+        elseif first_useful_token.text == "dump-cache" or
+               (first_useful_token.text == "run" and
+                tokens[first_useful_token_index + 1].text == "mv") then
```

Comment:
I don't feel like it is worth it special casing `mv`. There's a bunch of shell commands where completing would make sense but I don't think we want to go down that rabbit hole do we?

Reply:
Changed to complete files for all arguments to `run` since the second.

Reply:
That makes sense.